Ginger/Ginger/UserControlsLib/UCElementDetails.xaml.cs

```diff
@@ -592,6 +592,19 @@ private List<ComboEnumItem> GetPlatformLocatByList()
             return locateByComboItemList;
         }
 
+        private List<ComboEnumItem> GetAllElementlist()
+        {
+            List<ComboEnumItem> Allelementlist = new List<ComboEnumItem>();
+            List<ElementInfo> MappedElementlist = mSelectedPOM != null ? mSelectedPOM.MappedUIElements.ToList() : new List<ElementInfo>();
+            foreach(var Elem in MappedElementlist)
```

Comment:
Not to iterate mapped elements for bindind

Reply:
for now custom friendly locator option is disable so we are not showing drop down currently